src/cli/setup_mode.rs

```diff
@@ -31,6 +31,12 @@ pub fn main() -> Result<()> {
                 .long("verbose")
                 .help("Enable verbose output"),
         )
+        .arg(
+            Arg::with_name("quiet")
+                .short("q")
+                .long("quiet")
+                .help("Disable progress output"),
+        )
```

Comment:
Would it make sense if quiet were to conflict with verbose?

Reply:
| quiet | verbose | comment |
|--------|------------|---------------|
| Y      | Y           | does not really makes sense: the purpose of quiet (less output) is the opposite of verbose (more output) |
| Y      | N           | expected usage for quiet |
| N      | Y           | expected usage for verbose |
| N      | N           | default usage |

It would indeed make sense if quiet and verbose were conflicting options =)